src/site/content/en/lighthouse-pwa/viewport/index.md

```diff
@@ -1,57 +1,66 @@
 
 <figure class="w-figure">
-  <img class="w-screenshot w-screenshot--filled" src="viewport.png" alt="Lighthouse audit shows page is missing a viewport">
-  <figcaption class="w-figcaption">
-    Page is missing a viewport.
-  </figcaption>
+  <img class="w-screenshot w-screenshot" src="viewport.png" alt="Lighthouse audit shows page is missing a viewport">
 </figure>
 
-## How this audit fails
+A page fails the audit unless all of these conditions are met:
+- The document's `<head>` contains a `<meta name="viewport">` tag.
+- The viewport meta tag contains a `content` attribute.
+- The `content` attribute's value includes the text `width=`.
 
-Lighthouse checks that there's a `<meta name="viewport">` tag in the `<head>`
-of the document. It also checks that the node contains a `content` attribute
-and that the value of this attribute contains the text `width=`. However,
-it does not check that `width` equals `device-width`. Lighthouse also does not
-check for a `initial-scale` key-value pair.
+Lighthouse _doesn't_ check that `width` equals `device-width`. It also doesn't
+check for an `initial-scale` key-value pair. However, you still need to include
+both for your page to render correctly on mobile devices.
 
 {% include 'content/lighthouse-pwa/scoring.njk' %}
 
-## Recommendations
+## How to add a viewport meta tag
 
-Add a viewport `<meta>` tag in the `<head>` of your HTML:
+Add a viewport `<meta>` tag with the appropriate key-value pairs to the `<head>`
+of your page:
 
-```html
-<head>
-  ...
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  ...
-</head>
+```html/4
+<!doctype html>
+<html lang="en">
+  <head>
+    …
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    …
+  </head>
+  <body>
+    …
+  </body>
+</html>
 ```
 
-The `width=device-width` key-value pair sets the width of the viewport to
-the width of the device. The `initial-scale=1` key-value pair sets the initial
-zoom level when visiting the page.
-
-## Why you should include a viewport
-
-Setting the viewport enables you to control the width and scaling of the viewport.
-Check out the following links to learn more:
-
-- [Configure the Viewport](/https://developers.google.com/speed/docs/insights/ConfigureViewport)
-- [Set the Viewport](https://developers.google.com/web/fundamentals/design-and-ux/responsive/#set-the-viewport)
+Here's what each key-value pair does:
+- `width=device-width` sets the width of the viewport to the width of the device.
+- `initial-scale=1` sets the initial zoom level when the user visits the page.
 
-## More information
+## Resources
 
-[Page doesn't have a viewport audit source](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/viewport.js)
+- [Source code for **Has a `<meta name="viewport">` tag with `width` or `initial-scale`** audit](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/viewport.js)
```

Comment:
I believe we said we wanted to just use `Source code` and omit the rest. We can just put this guidance into a handbook doc and do a dedicated PR to update them all later.

Reply:
Sorry, I don't remember this convo. In an exchange with Paul, we settled on "Source code for xxx audit," which seems more descriptive to me. Otherwise, how do you know what the source code is of? This page? The audit?